lib/samples/find_address_with_reverse_geocode/find_address_with_reverse_geocode_sample.dart

```diff
@@ -47,158 +51,78 @@ class _FindAddressWithReverseGeocodeSampleState
-          child: Padding(
-            padding: const EdgeInsets.all(10),
-            child: buildServiceControls(context),
-          ),
-        ),
-      ),
       resizeToAvoidBottomInset: false,
       body: Stack(
         children: [
           ArcGISMapView(
             controllerProvider: () => _mapViewController,
             onMapViewReady: onMapViewReady,
-            onTap: onTap,
-          ),
-          Visibility(
-            visible: _ready,
-            child: SafeArea(
-              child: Container(
-                margin: const EdgeInsets.all(10),
-                child: buildSearchControls(context),
-              ),
-            ),
+            onTap: _ready ? onTap : null,
           ),
         ],
       ),
     );
   }
 
-  Widget buildServiceControls(BuildContext context) {
-    return Column(children: [
-      const Text('Geocode Service'),
-      RadioListTile<LocatorTask>(
-        title: const Text('World'),
-        value: _worldLocatorTask,
-        groupValue: _locatorTask,
-        onChanged: onServiceChanged,
-      ),
-      RadioListTile<LocatorTask>(
-        title: const Text('San Diego'),
-        value: _sanDiegoLocatorTask,
-        groupValue: _locatorTask,
-        onChanged: onServiceChanged,
-      ),
-    ]);
-  }
-
-  Widget buildSearchControls(BuildContext context) {
-    return Row(
-      children: [
-        Expanded(
-          child: Container(
-            padding: const EdgeInsets.symmetric(horizontal: 10),
-            color: Colors.white,
-            child: TextField(
-              decoration: const InputDecoration(
-                hintText: 'Search...',
-              ),
-              onSubmitted: onSearchSubmitted,
-            ),
-          ),
-        ),
-        IconButton(
-          icon: const Icon(
-            Icons.list,
-            color: Colors.white,
-          ),
-          onPressed: () => _scaffoldStateKey.currentState!.openEndDrawer(),
-        ),
-      ],
-    );
-  }
-
   void onMapViewReady() async {
-    final map = ArcGISMap.withBasemapStyle(BasemapStyle.arcGISImagery);
+    final map = ArcGISMap.withBasemapStyle(BasemapStyle.arcGISTopographic);
 
     final image = await ArcGISImage.fromAsset('assets/pin_circle_red.png');
     final pictureMarkerSymbol = PictureMarkerSymbol.withImage(image);
     pictureMarkerSymbol.width = 35;
     pictureMarkerSymbol.height = 35;
     pictureMarkerSymbol.offsetY = pictureMarkerSymbol.height / 2;
+
     _graphicsOverlay.renderer = SimpleRenderer(symbol: pictureMarkerSymbol);
-    _graphicsOverlay.graphics.add(_graphic);
     _mapViewController.graphicsOverlays.add(_graphicsOverlay);
 
     _mapViewController.arcGISMap = map;
-
-    _geocodeParameters.minScore = 75;
-    _geocodeParameters.resultAttributeNames
-        .addAll(['Place_addr', 'Match_addr']);
+    _mapViewController.setViewpoint(_initialViewpoint);
 
     await Future.wait([
       _worldLocatorTask.load(),
-      _sanDiegoLocatorTask.load(),
     ]);
```

Comment:
The `Future.wait([])` thing was just because there were two Futures to wait on. Since we're down to one, we don't need `Future.wait([])` any more -- it can just be `await _worldLocatorTask.load();`